test/lint/lint-files.py

```diff
@@ -98,7 +98,7 @@ def check_source_filenames() -> int:
     for filename in filenames:
         if not filename_regex.match(filename) and not filename_exception_regex.match(filename):
             print(
-                f"""File {repr(filename)} does not not match the allowed source filename regexp ('{ALLOWED_SOURCE_FILENAME_REGEXP}'), or the exception regexp ({ALLOWED_SOURCE_FILENAME_EXCEPTION_REGEXP})."""
+                f"""File {repr(filename)} does not match the allowed source filename regexp ('{ALLOWED_SOURCE_FILENAME_REGEXP}'), or the exception regexp ({ALLOWED_SOURCE_FILENAME_EXCEPTION_REGEXP})."""
```

Comment:
these are not yet fixed upstream: https://github.com/bitcoin/bitcoin/blob/master/test/lint/lint-files.py#L94 

would you like to submit this patch there? it will then make it into elements later during merging

Reply:
Thanks @tianzedavid